src/integration-tests/introspector/introspectTest.sh

```diff
@@ -120,6 +120,7 @@ export ADMIN_NAME=${ADMIN_NAME:-"admin-server"}
 export ADMIN_PORT=${ADMIN_PROT:-7001}
 export MANAGED_SERVER_NAME_BASE=${MANAGED_SERVER_NAME_BASE:-"managed-server"}
 export DOMAIN_NAME=${DOMAIN_NAME:-"base_domain"}
+export ADMINISTRATION_PORT=${ADMINISTRATION_PORT:-7099}
```

Comment:
No file in this pull uses 'ADMINISTRATION_PORT'.  I suspect this is dead code.

Reply:
createDomain script uses it to set the AdministrationPort..